.github/workflows/main.yml

```diff
@@ -0,0 +1,65 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: true
+          # we need full git history for article dates
+          fetch-depth: 0
+
+      - name: Setup Haxe
+        uses: krdlab/setup-haxe@v1
+        with:
+          haxe-version: 4.3.5
+
+      # we are using this version in order to build vscode-textmate
+      - name: Setup Node
+        uses: actions/setup-node@v1
+
+      - name: Install
+        # at some point all npm needs to be in package.json
+        run: |
+          npm install
+          npm install -g less@2.7
+          npm install -g less-plugin-clean-css@1.5
+          haxelib install CodeCookBook-neko.hxml --always --quiet
+          haxelib install highlighting.hxml --always --quiet
+          haxelib list
+
+      - name: Setup Pages
+        id: pages_config
+        uses: actions/configure-pages@v5
+
+      - name: Generate website
+        run: |
+          haxe CodeCookBook-neko.hxml
+          haxe highlighting.hxml
+        env:
+          GHP_ROOT: ${{steps.pages_config.outputs.base_url}}
```

Comment:
This should be removed. It doesn't seem to be used in this repository, and was what was triggering infinite redirections on nekovm.org

Reply:
Fixed.